### DISCUSSION TITLE
Netlify app with SvelteKit and PocketBase

### DISCUSSION THREAD

**Discussion by dsm-72** [pockethost/pockethost#188] [Q&A]
Hey!
I want to try and deploy an app made with SvelteKit and PocketBase.
I want to use Netlify but IDK if it is possible.
I was recommended PocketHost.
How would I do this (for someone unfamiliar)

**Comment by benallfree** [comment]
PocketHost will host the backend. Currently you need to host the front end somewhere else, but we are working on allowing earthing to be hosted with PocketHost. 

Netlify cannot run PocletBase backends, as far as I know. 

I hope that helps!
